tests/test_db_store.py

```diff
@@ -34,6 +34,20 @@ def test_latest_soil_moisture_empty_database(self):
         moisture = store.latest_soil_moisture()
         self.assertIsNone(moisture)
 
+    def test_retrieve_soil_moisture(self):
+        mock_cursor = mock.Mock()
+        store = db_store.SoilMoistureStore(mock_cursor)
+        timestamp1 = str(datetime.datetime(2016, 7, 23, 10, 51, 9, 928000))
+        timestamp2 = str(datetime.datetime(2016, 7, 23, 10, 52, 9, 928000))
+        mock_cursor.fetchall.return_value = [(timestamp1, 300),
+                                             (timestamp2, 400)]
+        soil_moisture_data = store.retrieve_soil_moisture()
+
+        self.assertEqual(soil_moisture_data[0].timestamp, timestamp1)
```

Comment:
Mm this and the above comment are interesting to me. The reason that I tend to prefer using ```timestamp``` variables is that for me my eyes tend to glaze over when I'm looking at ```datetime``` objects (or any "long" objects) and I think I'd find it difficult reading someone else's tests to figure out what's supposed to be matching up with what, whereas with named variables it's clear. I think I'd prefer using naming like ```string_timestamp1``` or something to clear up the ambiguity. I suppose another good option is to use simpler ```datetime``` values that are more readable, which I think might have already been your suggestion on a previous PR.